Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM node:13.0.1-alpine AS buildstep
+LABEL maintainer "RPS <devops@rpsgroup.com>"
+
+RUN mkdir -p /web
+WORKDIR /web
+
+COPY ./web/ /web/
+RUN yarn global add grunt-cli && \
+    yarn install && \
+    grunt
+
+FROM python:3.6
+
+RUN mkdir -p /glider-dac-status
+RUN mkdir -p /glider-dac-status/logs
```

Comment:
`mkdir -p` will create any nonexistent parent directories if they don't exist, so the call to `mkdir -p /glider-dac-status` is redundant.